Cython/Compiler/Nodes.py

```diff
@@ -8631,6 +8654,14 @@ def analyse_declarations(self, env):
                 self.import_star = 1
             else:
                 target.analyse_target_declaration(env)
+                if target.entry:
+                    if target.get_known_standard_library_import() is None:
+                        target.entry.known_standard_library_import = "%s.%s" % (
+                            self.module.module_name.value, name)
```

Comment:
This probably needs to be an `EncodedString`.
```suggestion
                        target.entry.known_standard_library_import = EncodedString("%s.%s" % (
                            self.module.module_name.value, name))
```